File: articles/ai-and-the-future-of-decision-making/c/hybrid_decision_score.c
#include <stdio.h>

double decision_profile(double human, double machine, double coordination, double transparency, double uncertainty, double accountability, double contestability, double equity) {
    return 0.16 * human + 0.16 * machine + 0.16 * coordination + 0.12 * transparency
        + 0.12 * uncertainty + 0.12 * accountability + 0.08 * contestability + 0.08 * equity;
}

int main(void) {
    printf("High-governance hybrid decision profile: %.4f\n",
           decision_profile(0.72, 0.74, 0.83, 0.79, 0.78, 0.82, 0.80, 0.76));
    return 0;
}

File: articles/ai-and-the-future-of-decision-making/fortran/hybrid_decision_score.f90
program hybrid_decision_score
  implicit none
  real :: score

  score = 0.16*0.72 + 0.16*0.74 + 0.16*0.83 + 0.12*0.79 + &
          0.12*0.78 + 0.12*0.82 + 0.08*0.80 + 0.08*0.76

  print *, "High-governance hybrid decision profile:", score
end program hybrid_decision_score
